proposals/0522-or-patterns.rst

```diff
@@ -62,7 +62,7 @@ function we get
 
     stringOfT :: T -> Maybe String
     stringOfT (T1 s)        = Just s
-    stringOfT (one of T2{}, T3{}) = Nothing
+    stringOfT (T2{} ; T3{}) = Nothing
```

Comment:
```suggestion
    stringOfT (T2{}; T3{}) = Nothing
```

Reply:
Both of them are valid syntax.

Reply:
I know, but the rest of the text has no space before `;`.